src/ray/common/id.h

```diff
@@ -158,12 +164,35 @@ class TaskID : public BaseID<TaskID> {
 
   static TaskID FromRandom() = delete;
 
-  // TODO(qwang): Comments
+  /// \return The ID generated for driver task.
+  static TaskID ForDriverTask();
+
+  /// Creates a TaskID for actor creation task.
+  ///
+  /// \param actor_id The ID of the actor to which this creation task belongs.
+  ///
+  /// \return The ID of the actor creation task.
   static TaskID ForActorCreationTask(const ActorID &actor_id);
 
-  static TaskID ForActorTask(const ActorID &actor_id);
+  /// Creates a TaskID for actor task.
+  ///
+  /// \param job_id The ID of the job to which this task belongs.
+  /// \param parent_task_id The ID of the parent task which submitted this task.
+  /// \param parent_task_counter The number of the tasks which the parent task submitted.
```

Comment:
`A count of the number of tasks submitted by the parent task before this one.` (copied from `common.proto`